trajectorytools/trajectories.py

```diff
@@ -488,7 +489,9 @@ def get_bouts(self, find_max_dict=None, find_min_dict=None):
         all_bouts = []
         speed = self.speed
         for focal in range(self.number_of_individuals):
-            bouts = find_bouts_individual(speed[:, focal])
+            bouts = find_bouts_individual(
```

Comment:
can you use the kwargs in the arguments so we make sure we do not exchange them by mistake?

Reply:
Done!